docs/src/main/sphinx/object-storage/legacy-azure.md

```diff
@@ -4,6 +4,13 @@ The {doc}`/connector/hive` can be configured to use [Azure Data Lake Storage
 (Gen2)](https://azure.microsoft.com/products/storage/data-lake-storage/). Trino
 supports Azure Blob File System (ABFS) to access data in ADLS Gen2.
 
+:::{warning}
+For the legacy implementation to work, the `fs.hadoop.enabled=true` property
+must be configured in your catalog configuration file. Upgrading to the [native
```

Comment:
Don't talk about it as an upgrade. Just call out that using this legacy support is not recommended and support will be removed. Use the native support instead,